src/databricks/labs/remorph/snow/databricks.py

```diff
@@ -359,7 +359,7 @@ class Databricks(org_databricks.Databricks):  #
     databricks = org_databricks.Databricks()
     NULL_ORDERING = "nulls_are_small"
 
-    class Generator(org_databricks.Databricks.Generator):
+    class Generator(org_databricks.Databricks.Generator):  # pylint: disable=too-many-public-methods
```

Comment:
Move this to the beginning of the file.

@nfx need your override as while overloading the sqlglot functions requires this pylint to be disabled in the file.

Reply:
@sundarshankar89 moved it to the beginning